src/main/java/training/day2/HomeworkFromMonday.java

```diff
@@ -0,0 +1,37 @@
+package training.day2;
+
+import java.util.Scanner;
+public class HomeworkFromMonday {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int day = scanner.nextInt();
+
+        if(day >= 1 && day <= 7){
```

Comment:
When using case, such checking is not needed. Add "default" instead. The switch will check every possible case. If none of the cases is affirmative, then the default will execute. So, remove the if statement and add default.